### PR TITLE
fixes #3980 - Compute profiles not rendering Openstack attributes correctly

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
@GregSutcliffe, can you verify if this fix works for you.

**Comment by GregSutcliffe** [comment]
@isratrade sure, will test it on my Openstack instance when I get home tonight.

**Comment by GregSutcliffe** [comment]
@isratrade confirmed this fixes the forms for me.

**Comment by GregSutcliffe** [comment]
@isratrade actually this only fixes the floating IP choice - can we do something similar for the security group? That field has the same problem...

**Comment by GregSutcliffe** [comment]
Also, we need an include_blank on these fields, I can't unset a floating IP network choice from a profile (or from a New Host)

File: app/views/compute_resources_vms/form/_openstack.html.erb [hunk 1]
```diff
@@ -12,6 +12,6 @@
 </div>
 <%= select_f f, :tenant_id, compute_resource.tenants, :id, :name, {}, :label => _('Tenant') %>
 
-<%= select_f f, :security_groups, compute_resource.security_groups, :name, :name, {}, :label => _("Security group")  %>
-<%= selectable_f f, :network, compute_resource.address_pools, { :prompt => "None" }, { :label => _("Floating IP network") } %>
+<%= select_f f, :security_group, compute_resource.security_groups, :name, :name, { :include_blank => _("None") }, :label => _("Security group")  %>
```
Comment (isratrade) [inline app/views/compute_resources_vms/form/_openstack.html.erb:15]: @ohadlevy, I switched this from `security_groups` to `security_group`

**Comment by GregSutcliffe** [comment]
Tests well here, :+1: to merge

**Comment by domcleal** [comment]
Merged as 88daf55 for Foreman 1.4.1, thanks @isratrade & @GregSutcliffe.

**Pull request closed without merging**